datahub-web-react/src/app/shared/ManageAccount.tsx

```diff
@@ -16,17 +18,43 @@ const defaultProps = {
 
 export const ManageAccount = ({ urn: _urn, pictureLink: _pictureLink }: Props) => {
     const entityRegistry = useEntityRegistry();
+
+    const isLoggedIn = checkAuthStatus();
+
+    const handleLogout = () => {
+        Cookies.remove('PLAY_SESSION');
```

Comment:
technically, this line isn't doing anything. the play cookie is http only, so js isn't able to read or write it

Reply:
This line is removed now.